ffxivbot/webapi.py

```diff
@@ -256,23 +256,23 @@ def webapi(req):
 
 def github_webhook(req):
     req_json = json.loads(req.body)
-    event_type = req.META.get("HTTP_X_GITHUB_EVENT")
+    event_type = req.META.get("HTTP_X_GITEA_EVENT") or req.META.get("HTTP_X_GOGS_EVENT") or req.META.get("HTTP_X_GITHUB_EVENT")

```

Comment:
Can you set `HTTP_X_GITHUB_EVENT` as default?

Reply:
> Can you set `HTTP_X_GITHUB_EVENT` as default?

OK, I have already deleted this.

the headers of gitea seems like

```
X-GitHub-Delivery: f6266f16-1bf3-46a5-9ea4-602e06ead473
X-GitHub-Event: push
X-Gogs-Delivery: f6266f16-1bf3-46a5-9ea4-602e06ead473
X-Gogs-Event: push
X-Gitea-Delivery: f6266f16-1bf3-46a5-9ea4-602e06ead473
X-Gitea-Event: push
```

so it's unnecessary to add this line.